src/log/Log.cc

```diff
@@ -178,6 +179,26 @@ void Log::stop_graylog()
   m_graylog.reset();
 }
 
+void Log::set_journald_level(int log, int crash)
+{
+  std::scoped_lock lock(m_flush_mutex);
+  m_journald_log = log;
+  m_journald_crash = crash;
+}
+
+void Log::start_journald()
+{
+  std::scoped_lock lock(m_flush_mutex);
+  if (! m_journald.get())
```

Comment:
nit, might want to enclose this branch in curly brackets for better readability.